### DISCUSSION TITLE
The packageManager digest is missing in the PR conversation

### DISCUSSION THREAD

**Discussion by MartinX3** [renovatebot/renovate#32511] [Request Help]
### How are you running Renovate?

A Mend.io-hosted app

### Please tell us more about your question or problem

The renovate bot adds a new hash via the PR but the PR description still shows the old behavior before the fix.

| Package | Change | Age | Adoption | Passing | Confidence |
|---|---|---|---|---|---|
| [pnpm](https://pnpm.io) ([source](https://redirect.github.com/pnpm/pnpm/tree/HEAD/pnpm)) | [`9.12.3+sha256.24235772cc4ac82a62627cd47f834c72667a2ce87799a846ec4e8e555e2d4b8b` -> `9.13.0`](https://renovatebot.com/diffs/npm/pnpm/9.12.3/9.13.0) | [![age](https://developer.mend.io/api/mc/badges/age/npm/pnpm/9.13.0?slim=true)](https://docs.renovatebot.com/merge-confidence/) | [![adoption](https://developer.mend.io/api/mc/badges/adoption/npm/pnpm/9.13.0?slim=true)](https://docs.renovatebot.com/merge-confidence/) | [![passing](https://developer.mend.io/api/mc/badges/compatibility/npm/pnpm/9.12.3/9.13.0?slim=true)](https://docs.renovatebot.com/merge-confidence/) | [![confidence](https://developer.mend.io/api/mc/badges/confidence/npm/pnpm/9.12.3/9.13.0?slim=true)](https://docs.renovatebot.com/merge-confidence/) |

![image](https://github.com/user-attachments/assets/8df6d881-58ba-44be-bfa0-d77cb5a46c97)

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment]
I think it should read 9.12.3 -> 9.13.0. Adding the hashes provides no value to the user. It's not like anyone pays attention to them in the PR body.
